spyder/plugins/explorer/tests/test_explorer.py

```diff
@@ -32,6 +43,15 @@ def project_explorer(qtbot):
     return widget
 
 
+@pytest.fixture
+def copy_path_file(qtbot):
+    """Setup Project Explorer widget."""
+    project_directory = getcwd_or_home()
```

Comment:
You should set this to the temporary directory (which you should also create here as part of the fixture, noted elsewhere) to avoid the test failures on our Azure CI (where the home dir is on a different drive as the temporary dir, causing the relative path test to fail) and better replicate "real-world" conditions in Spyder.

Reply:
I did that, Now at least the tests are passing I believe!

Reply:
Great!